### PR TITLE
[Edit] JS: Operators

### PR DESCRIPTION
**Pull request by Dusch4593**
### Description

Edited the [JavaScript: Operators](https://www.codecademy.com/resources/docs/javascript/operators) entry by mining out information from [this Codecademy article about Comparison and Logical operators](https://www.codecademy.com/article/fwd-js-comparison-logical). Included more examples and one Codebyte example snippet at the end.

### Type of Change

- Editing an existing entry (fixing a typo, bug, issues, etc)

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Comment by Dusch4593** [comment]
Please feel free to scrutinize the examples I wrote, especially the ones under "Logical Operators" and "Conditional Operator". 😄

File: content/javascript/concepts/operators/operators.md [hunk 1]
```diff
@@ -5,55 +5,68 @@ Subjects:
...
+
+An assignment operator assigns a value to its left operand based on the value of its right operand:
 
-An assignment operator assigns a value to its left operand based on the value of its right operand. Here are some of them:
+- `+=`: Adds and assigns a new value to a variable.
+- `-=`: Subtracts and assigns a new value to a variable.
+- `*=`: Multiplies and assigns a new value to a variable.
+- `/=`: Divides and assigns a new value to a variable.
+- `%=`: Assigns the returned remainder (modulo) as a new value to a variable.
+- `**=`:
```
Comment (SSwiniarski) [inline content/javascript/concepts/operators/operators.md:65]:
```suggestion
- `**=`: Assigns the left operand raised to the power of the right operand.
```

File: content/javascript/concepts/operators/operators.md [hunk 2]
```diff
@@ -68,12 +81,93 @@ console.log(number);
...
+
+console.log(isDuck);
+// Output: true
+
+const isBird = true;
+const isPlane = false;
+
+// OR Operator
+isDuck = isBird || isPlane;
+console.log(isDuck);
```
Comment (SSwiniarski) [inline content/javascript/concepts/operators/operators.md:143]:
```suggestion
console.log(isDuck);
// Output: true
```

File: content/javascript/concepts/operators/operators.md [hunk 3]
```diff
@@ -68,12 +81,93 @@ console.log(number);
...
+let isDuck = walksLikeADuck && talksLikeADuck;
+
+console.log(isDuck);
+// Output: true
+
+const isBird = true;
+const isPlane = false;
+
+// OR Operator
+isDuck = isBird || isPlane;
```
Comment (SSwiniarski) [inline content/javascript/concepts/operators/operators.md:142]:
```suggestion
isDuck = isBird || isPlane;

```

**Comment by Dusch4593** [comment]
@SSwiniarski Finished implementing your suggestions. 😄

**Review by SSwiniarski — APPROVED** [review]
@Dusch4593 Looks good for review two.

**Review by KTom101 — CHANGES_REQUESTED** [review]
@Dusch4593 I've made some comments on this.

File: content/javascript/concepts/operators/operators.md [hunk 4]
```diff
@@ -68,12 +81,95 @@ console.log(number);
...
-- `!=` not equal
-- `!==` not equal value or not equal type
-- `>` greater than
-- `<` less than
-- `>=` greater than or equal to
-- `<=` less than or equal to
-- `?` ternary operator
+These operators compare values and return a boolean value of `true` or `false`.
+
+- `==`: Returns `true` or `false` based on whether the value of two operands are equal.
```
Comment (HishamT) [inline content/javascript/concepts/operators/operators.md:86]: Do we want to put a warning that there is a difference between `==` and `=`? I feel like some people might confuse the two together.

**Review by HishamT — APPROVED** [review]
LGTM!

File: content/javascript/concepts/operators/operators.md [hunk 5]
```diff
@@ -68,12 +81,95 @@ console.log(number);
...
+- `<=`: Returns `true` or `false` based on whether the first value is less than or equal to the second value.
+
+### Example
+
+The following example showcases some of these comparison operators:
+
+```js
+let tenString = '10';
+let numberTen = 10;
+
```
[the remark this answers is not in the document]
Comment (Dusch4593) [reply content/javascript/concepts/operators/operators.md:102]: I agree! 👍🏻

File: content/javascript/concepts/operators/operators.md (hunk 4 above)
Comment (Dusch4593) [reply content/javascript/concepts/operators/operators.md:86]: @HishamT Excellent suggestion, I'll add a note! 😄

**Comment by Dusch4593** [comment]
@HishamT @KTom101 Thank you for your feedback. 😄 The changes were just implemented.

**Comment by Dusch4593** [comment]
Hey @KTom101 do the changes look good or is there anything I missed?

**Review by KTom101 — APPROVED** [review]
@Dusch4593 LGTM 👍

**Pull request merged**